portfolio/src/main/webapp/index.html

```diff
@@ -150,7 +154,21 @@ <h2>Slide Show</h2>
 <div id="connect">
   <h2>Connect with me</h2>
   <p>
-    Follow me here
+    Follow me here<br>
```

Comment:
This `<br>` shouldn't be necessary. If you need a little more space you can add `margin` to `.container` in css.

Reply:
Changes made